cli/js/40_testing.js

```diff
@@ -3,1353 +3,1374 @@
 // Do not use primordials because we do not want to depend on the __bootstrap
 // namespace.
 //
-// deno-lint-ignore-file prefer-primordials
-
-const core = globalThis.Deno.core;
-const ops = core.ops;
-
-const internals = globalThis.__bootstrap.internals;
-const {
-  setExitHandler,
-  Console,
-  serializePermissions,
-} = internals;
-
-const opSanitizerDelayResolveQueue = [];
-let hasSetOpSanitizerDelayMacrotask = false;
-
-// Even if every resource is closed by the end of a test, there can be a delay
-// until the pending ops have all finished. This function returns a promise
-// that resolves when it's (probably) fine to run the op sanitizer.
-//
-// This is implemented by adding a macrotask callback that runs after the
-// all ready async ops resolve, and the timer macrotask. Using just a macrotask
-// callback without delaying is sufficient, because when the macrotask callback
-// runs after async op dispatch, we know that all async ops that can currently
-// return `Poll::Ready` have done so, and have been dispatched to JS.
-//
-// Worker ops are an exception to this, because there is no way for the user to
-// await shutdown of the worker from the thread calling `worker.terminate()`.
-// Because of this, we give extra leeway for worker ops to complete, by waiting
-// for a whole millisecond if there are pending worker ops.
-function opSanitizerDelay(hasPendingWorkerOps) {
-  if (!hasSetOpSanitizerDelayMacrotask) {
-    core.setMacrotaskCallback(handleOpSanitizerDelayMacrotask);
-    hasSetOpSanitizerDelayMacrotask = true;
+// deno-lint-ignore-file
+{
```

Comment:
Why wrap it like this? It causes a huge diff for the reason I don't understand.

Reply:
Because `execute_script` runs this file in global scope unlike ESM and we don't want to leak the internal variables out.

Reply:
Oh right, I forgot that these are meant to be scripts, instead of modules.